.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .DS_Store
 src/.DS_Store
 src/_includes/.DS_Store
+.env
```

Comment:
We can remove this since we don't have a .env file, right?

Reply:
Yup, resolved in [13780f5](https://github.com/kindsys/digital-services-coalition/pull/23/commits/13780f51a05ab2c88e2c293a51ea03279538ef35)